components/dashboard/src/components/RepositoryFinder.tsx

```diff
@@ -68,11 +72,13 @@ export default function RepositoryFinder({
         [onChange, repos],
     );
 
-    // Resolve the selected context url & project id props to a suggestion entry
-    const selectedSuggestion = useMemo(() => {
+    const [selectedSuggestion, setSelectedSuggestion] = useState<SuggestedRepository | undefined>(undefined);
+
+    // Resolve the selected context url & configurationId id props to a suggestion entry
+    useEffect(() => {
```

Comment:
I am sad about having to use an effect here replacing the old memo, but it is the only way I found we can reliably skip updates.

Why would we want to skip updates? Because if you select repo A and then open the combobox again and search for repo B, we lose track and information about repo A.